test/unittest/dds/xtypes/dyndata_idl/types/data_test/DataTest.idl

```diff
@@ -0,0 +1,127 @@
+struct PrimitivesStruct
+{
+    boolean my_bool;
+    octet my_octet;
+    char my_char;
+    wchar my_wchar;
+    long my_long;
+    unsigned long my_ulong;
+    int8 my_int8;
+    uint8 my_uint8;
+    short my_short;
+    unsigned short my_ushort;
+    long long my_longlong;
+    unsigned long long my_ulonglong;
+    float my_float;
+    double my_double;
+    long double my_longdouble;
+};
+
+enum MyEnum
+{
+    A,
+    B,
+    C
+};
+
+@bit_bound(28)
+bitmask MyBitMask
+{
+    @position(0) flag0,
+    flag1,
+    flag2,
+    @position(26) flag26
+    // @position(5) flag5
```

Comment:
This file needs a bit of cleanup, I used it for testing random stuff.

Reply:
Also, if this type is going to be "all", "mega ultra complex" or whatever I would use that in the name.